/left/port: /dev/ttyUSB0
/right/port: /dev/ttyUSB1
/left/rpm: 200
/right/rpm: 200
/wheel_radius: 0.1
/robot_width: 0.5
